vignettes/safestats-vignette.Rmd

```diff
@@ -380,10 +368,20 @@ Replication attempts, however, are typically based on samples from a different p
 
 We show that both procedures are safe, that is, they do not lead to the tolerable type I error rate be exceeded, whereas classical $p$-values once again overreject. 
 -->
-### Optional continuation does not result in safe tests exceeding the tolerable $\alpha$-level
+
+### Two ways to optionally continue studies with safe testing
+Safe tests, as we will show below, do conserve the type I error rate under optional continuation. Optional continuation implies gathering more samples than was planned for because, for instance, (1) more funding came available and the experimenter wants to learn more, (2) the evidence looked promising, (3) a reviewer or editor urged the experimenter to collect more data, or (4) other researchers attempt to replicate the first finding. 
+
+A natural way to deal with the first three cases is by computing an s-value over the combined data set. This is permitted if the data come from the same population, and if the s-value used is a [_test martingale_](#martingale), which the s-values in this package are.
+
+Replication attempts, however, are typically based on samples from a different population. One way to deal with this is by multiplying the s-value computed from the original study with the s-value computed from the replication attempt. In this situation, the s-value formula for the replication study could also be _redesigned_ through the \code{design} function, for example when more information on nuisance parameters or effect size has become available for designing a more powerful test.
+
+We show that both procedures are safe, that is, they do not lead to the tolerable type I error rate be exceeded, whereas classical $p$-values once again overreject. 
+
+### a. Optional continuation by _extending_ the sample size does not result in safe tests exceeding the tolerable $\alpha$-level
```

Comment:
_extending_ the experiment does not ... 


- "sample size" is as we just increase n, but we want more data at the same time
- removing "size" leads to "sample" which can be ambigious as we are dealing with a paired-sample test. So I thought that "data" or "experiment" would be best.